ormar/queryset/field_accessor.py

```diff
@@ -268,22 +268,40 @@ def isnull(self, other: Any) -> FilterGroup:
         """
         return self._select_operator(op="isnull", other=other)
 
-    def asc(self) -> OrderAction:
+    def asc(self, nulls_ordering: Optional[NullsOrdering] = None) -> OrderAction:
```

Comment:
You should also be able to pass this param in `order_by` in queryset and querysetproxy for related models.

Reply:
> You should also be able to pass this param in `order_by` in queryset and querysetproxy for related models.

I do not understand this
How to determine this amount in `order_by()` and without `asc()` or `desc()`?

Reply:
In order by you can pass OrderActions or strings too. If you pass `sort_order` it's ascending, if you pass `-sort_order` it's descending. For nested models double underscore is required so like `main_model_relation_field__child_model_field`. It also supports putting a `-` before nested related model fields and then it's descending. The logic is starting in order_by where OrderActions are created from strings.